payments-model/src/test/java/com/stripe/android/repository/ConsumersApiServiceImplTest.kt

```diff
@@ -50,6 +50,9 @@ class ConsumersApiServiceImplTest {
             bodyPart("phone_number", "%2B15555555568"),
             bodyPart("country", "US"),
             bodyPart("locale", "en-US"),
+            bodyPart("amount", "1234"),
+            bodyPart("currency", "cad"),
+            bodyPart("financial_incentive%5Bpayment_intent%5D", "pi_123"),
```

Comment:
We started adding a `urlEncode` call to lots of these, which makes it much easier to read.